rules/potential.smk

```diff
@@ -156,21 +166,21 @@ rule local_land_cover:
 rule local_built_up_area:
     message: "Determine the built up area for administrative units in layer {wildcards.layer}."
     input:
-        "src/built_up_area.py",
-        rules.settlements.output.built_up,
-        rules.units.output
+        script = script_dir + "built_up_area.py",
+        built_up_area = rules.settlements.output.built_up,
+        units = rules.units.output
     output:
         "build/{layer}/built-up-areas.csv"
     conda: "../envs/default.yaml"
-    shell:
-        PYTHON_SCRIPT
+    script: "../scripts/built_up_area.py"
 
 
+# FIXME: where is eligibility_local.py??
```

Comment:
I deleted the file in this commit: 6f739a5606d8fd1b1d554f6872aaa1d2ad86bf28

Reply:
Likely the rule is obsolete. Have you searched in the entire repo for any use of its output?

Reply:
ok, I was just confused why it was there and wondered whether there was a reason I couldn't fathom that the rule was sticking around...